static/js/settings_user_groups.js

```diff
@@ -282,7 +282,7 @@ exports.set_up = function () {
     meta.loaded = true;
     exports.populate_user_groups();
 
-    $(".organization").on("submit", "form.admin-user-group-form", function (e) {
+    $(".organization").off("submit").on("submit", "form.admin-user-group-form", function (e) {
```

Comment:
It turns out what I really want is below, so I'll make those changes.  FYI @Kumar-Tarun 
~~~
 $(".organization form.admin-user-group-form").off('submit').on('submit', function (e) {